packages/wallet/frontend/src/pages/kyc.tsx

```diff
@@ -0,0 +1,74 @@
+import { HeaderLogo } from '@/components/HeaderLogo'
+import AuthLayout from '@/components/layouts/AuthLayout'
+import { userService } from '@/lib/api/user'
+import { NextPageWithLayout } from '@/lib/types/app'
+import { GetServerSideProps, InferGetServerSidePropsType } from 'next/types'
+import { useEffect } from 'react'
+
+type KYCPageProps = InferGetServerSidePropsType<typeof getServerSideProps>
+
+const KYCPage: NextPageWithLayout<KYCPageProps> = ({ url }) => {
+  // const [openDialog, closeDialog] = useDialog()
```

Comment:
do we need the comments?

Reply:
Commented out for now. We will keep/remove as needed once we can test the KYC flow.